### PR TITLE
DDCE-4358 : Fix a11y issues + remove beta banner

### PR DESCRIPTION
**Pull request by lmugnai**

### CODE REVIEW

**Comment by lmugnai** [comment]
I've created this ticket to address the removal of SecureGCMCipher
https://jira.tools.tax.service.gov.uk/browse/DDCE-4534

**Review by EdCope — CHANGES_REQUESTED** [review]
The only addition is putting https://github.com/hmrc/bc-passengers-frontend#accessibility-tests instructions to the Readme

File: app/views/govuk_wrapper.scala.html [hunk 1]
```diff
@@ -53,22 +53,6 @@
...
 }
 
-@feedbackBanner = {
-    @messages("feedback.before")
-    <a rel="noopener noreferrer" class='govuk-link' href="mailto:team-hoss2-aaaadxh2vg7pg2qf4idj7rmi2e@@hmrcdigital.slack.com">@messages("feedback.link")</a>
-    @messages("feedback.after")
-}
-
-@beforeContentBlock = {
-    @govukPhaseBanner(PhaseBanner(
```
Comment (EdCope) [inline app/views/govuk_wrapper.scala.html:63]: Please remove `govukPhaseBanner: GovukPhaseBanner` template argument as well as component
and related messages https://github.com/hmrc/home-office-immigration-status-frontend/blob/main/conf/messages#L1-L5

File: app/views/AccessibilityStatementPage.scala.html
Comment (EdCope) [inline app/views/AccessibilityStatementPage.scala.html:1]: Can we change [this](https://github.com/hmrc/home-office-immigration-status-frontend/pull/203/files#diff-3fd07b5b94198d897607ff65829b674743beded8373da8d035a03fbbcc461858L40) path  `<a href="/check-immigration-status">`

**Review by EdCope — COMMENTED** [review]
The only addition is putting https://github.com/hmrc/bc-passengers-frontend#accessibility-tests instructions to the Readme

File: app/views/AccessibilityStatementPage.scala.html
Comment (lmugnai) [reply app/views/AccessibilityStatementPage.scala.html:1]: Is this just so it will work in pre-prod envs?

Comment (lmugnai) [reply app/views/AccessibilityStatementPage.scala.html:1]: Also, what do you want the content of the link to be? currently: https://admin.tax.service.gov.uk/check-immigration-status

Comment (EdCope) [reply app/views/AccessibilityStatementPage.scala.html:1]: Yes just for pre-prod, I think the content can probably stay the same though(?)

Comment (Taiwo-AB) [reply app/views/AccessibilityStatementPage.scala.html:1]: Yh the content can be the same. As the route is part of the service, I feel we should be able it should work in all pre-prod envs.

Comment (lmugnai) [reply app/views/AccessibilityStatementPage.scala.html:1]: hmmm, I don't like the idea of having the link different to the content. I'm not really sure it needs changing. If we ever need to click on this link in any sort of testing capacity we can simply change the domain manually. I don't think we have any tests checking this either.

Comment (lmugnai) [reply app/views/AccessibilityStatementPage.scala.html:1]: the only other way we could address this is to have this url built from config. Which just seems a bit overkill.

Comment (Taiwo-AB) [reply app/views/AccessibilityStatementPage.scala.html:1]: Maybe the wording needs changing in that regard. Not a fan of hardcording urls within a service that doesn't work in any pre-prod envs.

Comment (Taiwo-AB) [reply app/views/AccessibilityStatementPage.scala.html:1]: Maybe we just need a content around the link that doesn't specify the prod url like for example [here](https://github.com/hmrc/pbik-frontend/blob/3d4e4fd45e4ad3fb79006ca383c3a6909eadbc58/conf/messages#L226)

Comment (Taiwo-AB) [reply app/views/AccessibilityStatementPage.scala.html:1]: Just had a look at the text, I don't think we need `, available at <a href="https://admin.tax.service.gov.uk/check-immigration-status">https://admin.tax.service.gov.uk/check-immigration-status</a></p>` I think the link should be `Check Immigration Status service`. what do you think?

Comment (lmugnai) [reply app/views/AccessibilityStatementPage.scala.html:1]: Yes, I feel that would be better. I'll pass this by @matthew-barker-hmrc in Jess' absence

Comment (Taiwo-AB) [reply app/views/AccessibilityStatementPage.scala.html:1]: Thanks

**Review by EdCope — APPROVED** [review]

**Pull request merged**